connector/fyle_integrations_platform_connector/apis/expense_custom_fields.py

```diff
@@ -40,6 +40,6 @@ def list_all(self):
         generator = self.connection.list_all(query_params)
 
```

Comment:
assign custom_fields = [] here, if else is not needed in for loop. 'data' key would be present every time